### DISCUSSION TITLE
Carrier Sense n/a

### DISCUSSION THREAD

**Discussion by m1027** [OpenCCU/OpenCCU#2406] [🇺🇸 Q&A]
### Describe the issue you are experiencing

Our setup with RaspberryMatic + HmIP-RFUSB + rpi3 stopped working.

- Our thermostats show a blinking antenna symbol (so no connection)
- In RaspberryMatic we see "carrier sense n/a"

### Describe the behavior you expected

We had a running system for nearly a year. After switching off the rpi3 with RaspberryMatic over the summer and switching it on, it stopped working.

### Steps to reproduce the issue

Always. See also details below.

### What is the version this bug report is based on?

3.69.7.20230506

### Which base platform are you running?

rpi3 (RaspberryPi3)

### Which HomeMatic/homematicIP radio module are you using?

RFUSB

### Anything in the logs that might be useful for us?

```txt
***** /var/log/messages *****
...
Aug 26 12:05:11 homematic-raspi local0.err ReGaHss: ERROR: CallGetValue failed; sVal = 0.000000 [ReadValue():iseDOMdpHSS.cpp:124]
Aug 26 12:05:11 homematic-raspi local0.warn ReGaHss: WARNING: XMLRPC 'getValue': rpcClient.isFault() failed (url: xmlrpc://127.0.0.1:32010, params: {"000A1F29934B16:1","LEVEL"}, result: [faultCode:-5,faultString:"Unknown Parameter value for value key: LEVEL"]) [CallXmlrpcMethod():iseXmlRpc.cpp:2618]
Aug 26 12:05:11 homematic-raspi local0.err ReGaHss: ERROR: XMLRPC 'getValue' call failed (interface: 1009, params: {"000A1F29934B16:1","LEVEL"}) [CallGetValue():iseXmlRpc.cpp:1445]
Aug 26 12:05:11 homematic-raspi local0.err ReGaHss: ERROR: CallGetValue failed; sVal = 0.000000 [ReadValue():iseDOMdpHSS.cpp:124]
Aug 26 12:05:11 homematic-raspi local0.warn ReGaHss: WARNING: XMLRPC 'getValue': rpcClient.isFault() failed (url: xmlrpc://127.0.0.1:32010, params: {"000A1F29934B16:1","WINDOW_STATE"}, result: [faultCode:-5,faultString:"Unknown Parameter value for value key: WINDOW_STATE"]) [CallXmlrpcMethod():iseXmlRpc.cpp:2618]
Aug 26 12:05:11 homematic-raspi local0.err ReGaHss: ERROR: XMLRPC 'getValue' call failed (interface: 1009, params: {"000A1F29934B16:1","WINDOW_STATE"}) [CallGetValue():iseXmlRpc.cpp:1445]
Aug 26 12:05:11 homematic-raspi local0.err ReGaHss: ERROR: CallGetValue failed; sVal = 0 [ReadValue():iseDOMdpHSS.cpp:124]
Aug 26 12:05:11 homematic-raspi local0.warn ReGaHss: WARNING: XMLRPC 'getValue': rpcClient.isFault() failed (url: xmlrpc://127.0.0.1:32010, params: {"000A1F29934B16:1","SET_POINT_MODE"}, result: [faultCode:-5,faultString:"Unknown Parameter value for value key: SET_POINT_MODE"]) [CallXmlrpcMethod():iseXmlRpc.cpp:2618]
Aug 26 12:05:11 homematic-raspi local0.err ReGaHss: ERROR: XMLRPC 'getValue' call failed (interface: 1009, params: {"000A1F29934B16:1","SET_POINT_MODE"}) [CallGetValue():iseXmlRpc.cpp:1445]
Aug 26 12:05:11 homematic-raspi local0.err ReGaHss: ERROR: CallGetValue failed; sVal = 0 [ReadValue():iseDOMdpHSS.cpp:124]
Aug 26 12:05:11 homematic-raspi local0.warn ReGaHss: WARNING: XMLRPC 'getValue': rpcClient.isFault() failed (url: xmlrpc://127.0.0.1:32010, params: {"000A1F29934B16:1","PARTY_TIME_START"}, result: [faultCode:-5,faultString:"Unknown Parameter value for value key: PARTY_TIME_START"]) [CallXmlrpcMethod():iseXmlRpc.cpp:2618]
Aug 26 12:05:11 homematic-raspi local0.err ReGaHss: ERROR: XMLRPC 'getValue' call failed (interface: 1009, params: {"000A1F29934B16:1","PARTY_TIME_START"}) [CallGetValue():iseXmlRpc.cpp:1445]
Aug 26 12:05:11 homematic-raspi local0.err ReGaHss: ERROR: CallGetValue failed; sVal =  [ReadValue():iseDOMdpHSS.cpp:124]
Aug 26 12:05:11 homematic-raspi local0.warn ReGaHss: WARNING: XMLRPC 'getValue': rpcClient.isFault() failed (url: xmlrpc://127.0.0.1:32010, params: {"000A1F29934B16:1","PARTY_TIME_END"}, result: [faultCode:-5,faultString:"Unknown Parameter value for value key: PARTY_TIME_END"]) [CallXmlrpcMethod():iseXmlRpc.cpp:2618]
Aug 26 12:05:11 homematic-raspi local0.err ReGaHss: ERROR: XMLRPC 'getValue' call failed (interface: 1009, params: {"000A1F29934B16:1","PARTY_TIME_END"}) [CallGetValue():iseXmlRpc.cpp:1445]
Aug 26 12:05:11 homematic-raspi local0.err ReGaHss: ERROR: CallGetValue failed; sVal =  [ReadValue():iseDOMdpHSS.cpp:124]
Aug 26 12:05:11 homematic-raspi local0.warn ReGaHss: WARNING: XMLRPC 'getValue': rpcClient.isFault() failed (url: xmlrpc://127.0.0.1:32010, params: {"000A1F29934B16:1","PARTY_SET_POINT_TEMPERATURE"}, result: [faultCode:-5,faultString:"Unknown Parameter value for value key: PARTY_SET_POINT_TEMPERATURE"]) [CallXmlrpcMethod():iseXmlRpc.cpp:2618]
Aug 26 12:05:11 homematic-raspi local0.err ReGaHss: ERROR: XMLRPC 'getValue' call failed (interface: 1009, params: {"000A1F29934B16:1","PARTY_SET_POINT_TEMPERATURE"}) [CallGetValue():iseXmlRpc.cpp:1445]
Aug 26 12:05:11 homematic-raspi local0.err ReGaHss: ERROR: CallGetValue failed; sVal = 0.000000 [ReadValue():iseDOMdpHSS.cpp:124]
Aug 26 12:05:11 homematic-raspi local0.warn ReGaHss: WARNING: XMLRPC 'getValue': rpcClient.isFault() failed (url: xmlrpc://127.0.0.1:32010, params: {"000A1F29934B16:1","PARTY_MODE"}, result: [faultCode:-5,faultString:"Unknown Parameter value for value key: PARTY_MODE"]) [CallXmlrpcMethod():iseXmlRpc.cpp:2618]
Aug 26 12:05:11 homematic-raspi local0.err ReGaHss: ERROR: XMLRPC 'getValue' call failed (interface: 1009, params: {"000A1F29934B16:1","PARTY_MODE"}) [CallGetValue():iseXmlRpc.cpp:1445]
Aug 26 12:05:11 homematic-raspi local0.err ReGaHss: ERROR: CallGetValue failed; sVal = 0 [ReadValue():iseDOMdpHSS.cpp:124]
Aug 26 12:05:11 homematic-raspi user.warn monit[1655]: 'hasUSB' status failed (1) -- no output
Aug 26 12:05:11 homematic-raspi user.err monit[1655]: 'eMMCLifeTimeCheck' status failed (1) -- no output
Aug 26 12:05:11 homematic-raspi user.err monit[1655]: 'rpi4usb3Check' status failed (1) -- no output
Aug 26 12:05:11 homematic-raspi user.err monit[1655]: Filesystem '/media/usb1' not mounted
Aug 26 12:05:11 homematic-raspi user.err monit[1655]: 'usb1' unable to read filesystem '/media/usb1' state
Aug 26 12:05:11 homematic-raspi user.info monit[1655]: 'usb1' trying to restart
Aug 26 12:05:29 homematic-raspi user.warn monit[1655]: 'hasUSB' status failed (1) -- no output
Aug 26 12:05:29 homematic-raspi user.err monit[1655]: Filesystem '/media/usb1' not mounted
Aug 26 12:05:29 homematic-raspi user.err monit[1655]: 'usb1' unable to read filesystem '/media/usb1' state
Aug 26 12:05:29 homematic-raspi user.info monit[1655]: 'usb1' trying to restart
Aug 26 12:05:47 homematic-raspi user.err monit[1655]: 'hasUSB' status failed (1) -- no output
```

### Additional information

We've been trying the following to fix the issue without success:

- Our system was working for several months.
- We've got an identical hardware setup (rpi3, RFUSB, same firmware) at a completely different location which works there.
- Firmware upgrade to the newest 3.69.7.20230626 and back to 3.69.7.20230506 did not help
- Moved the rpi3 3 meters away from other electronic devices.
- Switched off all wifi routers and devices entirely
- Used RFUSB with and withoutUSB cable (direclty plugged to rpi3)
- Tried all 4 different USB sockets in the rpi
- Rebooting (more often)
- The thermostats don't indicate to have low battery power
- Teaching in the devices again does not work as there is no connection
- We see duty cycle between 0% and 4% (but carrier sense always "n/a")
- When unplugging RFUSB then RaspberryMatic seems to notice that and issues a message that has to be confirmed so we assume the antenna stick is detected

Any ideas welcome...

[Sorry for multiple edits. Correct is RFUSB + rpi3.]

**Comment by m1027** [comment]
EDITs: You might have received the initial mail notification only with wrong information.
Correct is: rpi3 + RFUSB (not rpi4 and not RFMOD).

**Comment by m1027** [comment]
Well, can anybody explain what "carrier sense n/a" means?

**Comment by m1027** [comment]
A factory reset fixed the carrier sense: "n/a" is gone and I have "0%" back now.

When I apply my configuration backup though, carrier sense becomes again "n/a".

I'll try a factory reset again, will *not* apply my backup and then teach everything in again.

**Comment by MichaelN0815** [comment]
n/a means "not available". The CS is delivered by the HmIP-RFUSB. I can't imagine that mis-configuration could stop this data. More likely there is an issue with the soldering of the HmIP-RFUSB.

**Comment by m1027** [comment]
@MichaelN0815: Thanks, yeah, "not available" is the meaning of "n/a" but I meant it doesn't tell what could be the reason. Under what circumstances is "n/a" shown?

Currently I doubt that the HmIP-RFUSB is broken because it worked some weeks ago for a year already and is still detected by RaspberryMatic.

Hm, after the factory reset the "n/a" is gone. I have "carrier sense 0%" now.

As my backup causes the same trouble (see above) I am trying to teach-in the devices, but they are not detected.

Also tried: New batteries, thermostat factory reset.

**Reply by MichaelN0815** [reply]
N/a is shown when there is no Funkmodul detected which delivers this data. 
You could believe or not. Several people reported issues with the Funkmodul after month or years of operating and they are gone after re-soldering.

**Comment by m1027** [comment]
Okay, I got you. If this is indeed a known issue and re-soldering fixes it then this is indeed helpful and I'll try to replace the unit for test.

It is just a bit confusing, that "carrier sense n/a" is not shown anymore after the factory reset. Everything seems to be ok ("carrier sense: 0%").

**Comment by m1027** [comment]
@MichaelN0815: Thanks for pointing out soldering issues *even when everything had been working before*. Indeed, with a new RFUSB stick everything works.
